charts/traefik/2.0.0/questions.yaml

```diff
@@ -0,0 +1,980 @@
+              - variable: serviceType
+                label: "Connection Type"
+                description: "Connection Type between Traefik and the external service"
+                schema:
+                  hidden: false
+                  editable: true
+                  required: true
+                  default: "HTTP"
+                  type: string
+                  enum:
+                    - value: "HTTP"
+                      description: "HTTP"
+                    - value: "HTTPS"
+                      description: "HTTPS"
+              - variable: serviceKind
+                label: "Service Kind to proxy to"
+                schema:
+                  hidden: true
+                  editable: false
+                  type: string
+                  default: ""
+              - variable: entrypoint
+                label: "Select Entrypoint"
+                schema:
+                  type: string
+                  default: "websecure"
+                  required: true
+                  enum:
+                    - value: "websecure"
+                      description: "Websecure: HTTPS/TLS port 443"
+              - variable: host
+                label: "Domain Name"
+                required: true
+                schema:
+                  type: string
+              - variable: path
+                label: "path"
+                schema:
+                  type: string
+                  required: true
+                  hidden: false
+                  default: "/"
+              - variable: certType
+                label: "Select Certificate Type"
+                schema:
+                  type: string
+                  default: "selfsigned"
+                  enum:
+                    - value: ""
+                      description: "No Encryption/TLS/Certificates"
+                    - value: "selfsigned"
+                      description: "Self-Signed Certificate"
+                    - value: "ixcert"
+                      description: "TrueNAS SCALE Certificate"
+              - variable: certificate
+                label: "Select TrueNAS SCALE Certificate"
+                schema:
+                  type: int
+                  show_if: [["certType", "=", "ixcert"]]
+                  $ref:
+                    - "definitions/certificate"
+              - variable: authForwardURL
+                label: "Forward Authentication URL"
+                schema:
+                  type: string
+                  default: ""
+
+  - variable: warning
+    label: "This App binds to the same ports as TrueNAS SCALE UI: 80 and 443"
+    description: "Please be sure to set the UI to a different port or adapter"
+    group: "WARNING"
+    schema:
+        type: dict
+        attrs:
+          - variable: confirm-warning
+            description: "Check this to confirm you have made precautions against conflicts on port 80 and 443"
+            label: "I understand above warning and its consequences"
```

Comment:
```suggestion
    label: "This App binds to the same ports as the defaults for TrueNAS SCALE UI (80 and 443). This means you may not be able to access the TrueNAS SCALE UI after starting the App."
    description: "Please be sure to set the UI to a different port or adapter"
    group: "WARNING"
    schema:
        type: dict
        attrs:
          - variable: confirm-warning
            description: "Check this to confirm you have made precautions against conflicts on port 80 and 443, such as running the TrueNAS SCALE UI on a different port or adapter."
            label: "I understand above warning and its consequences"
```

Hopefully this provides a little bit more guidance. I wasn't able to see the description text ("Please be sure to set the UI to a different port or adapter") in the UI anywhere, which I guess was the source of my confusion.

Reply:
![image](https://user-images.githubusercontent.com/47820033/111160965-49d82e80-85a3-11eb-8274-6e0cac5077d8.png)
It's in the help text!

Reply:
@stavros-k I'm actually quite fine with tripple-double-ensure making-sure people know what they might run into...